soh/soh/SohMenuBar.cpp

```diff
@@ -639,6 +639,9 @@ void DrawEnhancementsMenu() {
                 UIWidgets::Tooltip("Adds a prompt to equip newly-obtained swords, shields and tunics");
                 UIWidgets::PaddedEnhancementCheckbox("Better Owl", CVAR_ENHANCEMENT("BetterOwl"), true, false);
                 UIWidgets::Tooltip("The default response to Kaepora Gaebora is always that you understood what he said");
+                UIWidgets::PaddedEnhancementCheckbox("Bridge Open At Night", CVAR_ENHANCEMENT("BridgeOpenAtNight"), true, false);
+                UIWidgets::Tooltip("Makes the bridge to Hyrule Market stay open at night.");
+                const bool bBridgeOpenAtNightEnabled = CVarGetInteger(CVAR_ENHANCEMENT("BridgeOpenAtNight"), 0);
                 UIWidgets::PaddedEnhancementCheckbox("Exit Market at Night", CVAR_ENHANCEMENT("MarketSneak"), true, false);
```

Comment:
Thanks for looking through this! Looks like the force enable got lost in one of the merge conflicts resolutions. This was setup as a separate option because I believe leaving the bridge open is viewed, generally, as a more consequential change routing-wise than market-sneak. Tying the two together would likely result in the removal of market-sneak from settings presets, and I'd personally rather scrap this than do so.

I'm on holiday for the next week, so won't get to this for a bit. Also not sure I'll be able to handle the audio issue in a hook, but I see where it's coming from.

Reply:
If there is a genuine use case for having 1 but not the other, which splitting the presets implies, then they are fine being separate. I just didn't imagine one, especially as in theory you can always just wait for daytime. 

This should be discussed among the other devs IMO to get people's opinions on things, but that can wait until next week. Enjoy your holiday, I can do other things in the mean time, and bring it up in dev chat when you get back.